openconnect cannot be built with libressl

When libressl set as provider openconnect fails to build with message 

```
checking for known-broken versions of OpenSSL... yes
configure: error: LibreSSL does not support Cisco DTLS.
Build with OpenSSL or GnuTLS instead.
```

[1522683103-install-net-apps_openconnect-7.08:0::exony.out.txt](https://github.com/ony/exony/files/1876815/1522683103-install-net-apps_openconnect-7.08.0.exony.out.txt)

I suggest to remove libressl from providers list for clarity so user can configure first instead of facing build fail.

--- a/packages/net-apps/openconnect/openconnect-7.08.exheres-0
+++ b/packages/net-apps/openconnect/openconnect-7.08.exheres-0
@@ -16,7 +16,7 @@ LICENCES="LGPL-2.1"
 SLOT="0"
 PLATFORMS="~amd64 ~armv7"
 MYOPTIONS="
-( providers: gnutls libressl openssl ) [[ number-selected = exactly-one ]]
+( providers: gnutls openssl ) [[ number-selected = exactly-one ]]
 "
 
 DEPENDENCIES="

